### ISSUE TITLE
Refactor CSS in `src/screens/OrganizationPeople/OrganizationPeople.tsx`

### ISSUE THREAD

**Issue by palisadoes** [PalisadoesFoundation/talawa-admin#2890]
# Refactor CSS in this file

To improve the UI/UX of Talawa-Admin we have been working on:
1. making the application more suitable to color blind users
2. streamlining all CSS into a single global file

This file contains embedded CSS references that should be completely moved to:
- `src/style/app.module.css`

## Additional information:
1. The foundational work for this was completed in this PR:
    - https://github.com/PalisadoesFoundation/talawa-admin/pull/2466
2. The desired single CSS file is:
    - `src/style/app.module.css`

## The challenge:
The goal is for this file to use this new design pattern. This means that:
1. All embedded CSS in this file must be merged into the global CSS file using pre-existing CSS.
1. All embedded CSS in this file must be removed

**Comment by Coolsheru2526** [comment]
@palisadoes can I work on this issue?

**Comment by palisadoes** [comment]
> @palisadoes can I work on this issue?

Our policy is to assign no more than two issues to each contributor across all repositories. This way everyone gets a chance to participate in the projects. We sometimes give exceptions for more urgent cases and sometimes we lose track, but the policy stands. You have reached your limit, please wait until your existing issues are closed before requesting more issues. You could unassign yourself from one of the other issues too.

Please don't use the tactic of asking to be assigned to more than 2 issues at a time. Our policy is to give everyone a chance and if you are first in line you will be assigned.

**Comment by aadhil2k4** [comment]
@palisadoes Can I work on this?

**Comment by AceHunterr** [comment]
Can I work on this issue?

**Comment by AceHunterr** [comment]
@palisadoes we have sx prop for styling inside the react components like this:
![original](https://github.com/user-attachments/assets/4d3be0d6-5628-41b8-8d0c-7e85abdda1b4)

For better consistency since we are already importing the styles and applying the classes... we can also apply the root element vars to the components ensuring further consistency like this:

![after change](https://github.com/user-attachments/assets/5e3ce8e3-55b1-47c6-ac27-0f806c73b93a)

I just wanted to ask this how can I test for these styled changes that are being made? Like after running the npm run serve where can I look for them?

**Comment by AceHunterr** [comment]
Actually I found that after creating the organisation then I could see the file....

Our original style are like this : 
![1st](https://github.com/user-attachments/assets/ec3d9da6-2134-48ef-ace8-83296ede89f2)
![2nd](https://github.com/user-attachments/assets/3ba50591-4f88-4de6-bf46-2cfba407afd5)

and if we use the global style variable then the desired output is the same:

![3rd](https://github.com/user-attachments/assets/e62ff115-e463-4b25-a7e8-fd131da35b6e)
![4th](https://github.com/user-attachments/assets/65cd1b54-d786-4d09-af2e-2459c8260a4a)


I cross checked that the effect is not being overwritten somewhere else by giving a random color like this:
![5th](https://github.com/user-attachments/assets/437bb1c5-b0b1-4939-873e-4b7c748b93b6)
![6th](https://github.com/user-attachments/assets/737709b3-9b00-406f-91c6-9b6894bc528d)


So shall I replace the sx prop hardcoded colours with global CSS vars?

**Comment by palisadoes** [comment]
> Actually I found that after creating the organisation then I could see the file....
> 
> Our original style are like this : ![1st](https://private-user-images.githubusercontent.com/88583647/398972358-ec3d9da6-2134-48ef-ace8-83296ede89f2.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUzMjk4OTQsIm5iZiI6MTczNTMyOTU5NCwicGF0aCI6Ii84ODU4MzY0Ny8zOTg5NzIzNTgtZWMzZDlkYTYtMjEzNC00OGVmLWFjZTgtODMyOTZlZGU4OWYyLlBORz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjI3VDE5NTk1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI1YmM1MDFkYWY0MzExNDE2NDMzMzg2MWZiMDRkYmEwMDM4OTcxMTY1MmZlNmRkMjMyMjFkYjY4ZDUwMWUxZDAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.mOpwDBGoYn_qTwprbWE3KHZ0YI637OW-KAxFW4exESk) ![2nd](https://private-user-images.githubusercontent.com/88583647/398972365-3ba50591-4f88-4de6-bf46-2cfba407afd5.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUzMjk4OTQsIm5iZiI6MTczNTMyOTU5NCwicGF0aCI6Ii84ODU4MzY0Ny8zOTg5NzIzNjUtM2JhNTA1OTEtNGY4OC00ZGU2LWJmNDYtMmNmYmE0MDdhZmQ1LlBORz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjI3VDE5NTk1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ3Mzk3ZjJhZDE1Mjk1YjYyYjlhZjJmY2Y1OWVhMDlhNjFjOWJjN2Q2NTI3N2FlNzBiODgyZjJlODJiYjcwZTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.yKix5zTbwKZxJTZhuzO9WmcH-aAwuvy7ROQfWR_4frc)
> 
> and if we use the global style variable then the desired output is the same:
> 
> ![3rd](https://private-user-images.githubusercontent.com/88583647/398972462-e62ff115-e463-4b25-a7e8-fd131da35b6e.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUzMjk4OTQsIm5iZiI6MTczNTMyOTU5NCwicGF0aCI6Ii84ODU4MzY0Ny8zOTg5NzI0NjItZTYyZmYxMTUtZTQ2My00YjI1LWE3ZTgtZmQxMzFkYTM1YjZlLlBORz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjI3VDE5NTk1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTYzMzNlMWQ5YjU3ODhjNjM4YzgwNTIxMWQyY2E3NjU3YWNlMWRmNDA5NzNhNTMzODc2MDZmYzExZDQwZDNhOTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0._6NQqphHZT7px7clUG3hzW9v2yiPs0mu-VpHMgE3G-U) ![4th](https://private-user-images.githubusercontent.com/88583647/398972467-65cd1b54-d786-4d09-af2e-2459c8260a4a.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUzMjk4OTQsIm5iZiI6MTczNTMyOTU5NCwicGF0aCI6Ii84ODU4MzY0Ny8zOTg5NzI0NjctNjVjZDFiNTQtZDc4Ni00ZDA5LWFmMmUtMjQ1OWM4MjYwYTRhLlBORz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjI3VDE5NTk1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI1OTdiNDcyNDg3ODkwNmI3ZDQxOWE5ZmI0MDFmYjk5NGFkNmM2MTgzYmEwYmMyYmMzYWM1YzY2M2VlYTJlZjcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.tVVA3K_uCl93x5n1wLLhs0XjQ7CnBM69sDQ_CoAxeZ4)
> 
> I cross checked that the effect is not being overwritten somewhere else by giving a random color like this: ![5th](https://private-user-images.githubusercontent.com/88583647/398972645-437bb1c5-b0b1-4939-873e-4b7c748b93b6.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUzMjk4OTQsIm5iZiI6MTczNTMyOTU5NCwicGF0aCI6Ii84ODU4MzY0Ny8zOTg5NzI2NDUtNDM3YmIxYzUtYjBiMS00OTM5LTg3M2UtNGI3Yzc0OGI5M2I2LlBORz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjI3VDE5NTk1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY3YzQwYzcyODFhZmNhODc4M2RkYmZhZTZmOWE2MTU2NGU1ZDgyZDY1YWYzNDc0NGFiMzM1ZWRjZmUzYzYyMDMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.dq86iWGobGWrDRijZyrf9UW_0Q765LH6JLR6LaIwDNk) ![6th](https://private-user-images.githubusercontent.com/88583647/398972650-737709b3-9b00-406f-91c6-9b6894bc528d.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUzMjk4OTQsIm5iZiI6MTczNTMyOTU5NCwicGF0aCI6Ii84ODU4MzY0Ny8zOTg5NzI2NTAtNzM3NzA5YjMtOWIwMC00MDZmLTkxYzYtOWI2ODk0YmM1MjhkLlBORz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjI3VDE5NTk1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY5NmJkNWRiOTFlOWJmYTU3ODNjOGNhOWMzM2EyNzNlMjBjMzMzOTE3OWNjNjJmMzY3NGZhNmEyNTkwZTM1ODgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0._TkoOpNhvGDEYwOyyW76mdjJFFGoVWIpVfX2VfBmQ-s)
> 
> So shall I replace the sx prop hardcoded colours with global CSS vars?

Did you replace the hard coded values? I didn't check

**Comment by palisadoes** [comment]
Also the blue header is too bold. Use the work of he reference PR to select a color

**Comment by AceHunterr** [comment]
> > Actually I found that after creating the organisation then I could see the file....
> > Our original style are like this : ![1st](https://private-user-images.githubusercontent.com/88583647/398972358-ec3d9da6-2134-48ef-ace8-83296ede89f2.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUzMjk4OTQsIm5iZiI6MTczNTMyOTU5NCwicGF0aCI6Ii84ODU4MzY0Ny8zOTg5NzIzNTgtZWMzZDlkYTYtMjEzNC00OGVmLWFjZTgtODMyOTZlZGU4OWYyLlBORz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjI3VDE5NTk1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI1YmM1MDFkYWY0MzExNDE2NDMzMzg2MWZiMDRkYmEwMDM4OTcxMTY1MmZlNmRkMjMyMjFkYjY4ZDUwMWUxZDAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.mOpwDBGoYn_qTwprbWE3KHZ0YI637OW-KAxFW4exESk) ![2nd](https://private-user-images.githubusercontent.com/88583647/398972365-3ba50591-4f88-4de6-bf46-2cfba407afd5.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUzMjk4OTQsIm5iZiI6MTczNTMyOTU5NCwicGF0aCI6Ii84ODU4MzY0Ny8zOTg5NzIzNjUtM2JhNTA1OTEtNGY4OC00ZGU2LWJmNDYtMmNmYmE0MDdhZmQ1LlBORz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjI3VDE5NTk1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ3Mzk3ZjJhZDE1Mjk1YjYyYjlhZjJmY2Y1OWVhMDlhNjFjOWJjN2Q2NTI3N2FlNzBiODgyZjJlODJiYjcwZTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.yKix5zTbwKZxJTZhuzO9WmcH-aAwuvy7ROQfWR_4frc)
> > and if we use the global style variable then the desired output is the same:
> > ![3rd](https://private-user-images.githubusercontent.com/88583647/398972462-e62ff115-e463-4b25-a7e8-fd131da35b6e.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUzMjk4OTQsIm5iZiI6MTczNTMyOTU5NCwicGF0aCI6Ii84ODU4MzY0Ny8zOTg5NzI0NjItZTYyZmYxMTUtZTQ2My00YjI1LWE3ZTgtZmQxMzFkYTM1YjZlLlBORz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjI3VDE5NTk1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTYzMzNlMWQ5YjU3ODhjNjM4YzgwNTIxMWQyY2E3NjU3YWNlMWRmNDA5NzNhNTMzODc2MDZmYzExZDQwZDNhOTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0._6NQqphHZT7px7clUG3hzW9v2yiPs0mu-VpHMgE3G-U) ![4th](https://private-user-images.githubusercontent.com/88583647/398972467-65cd1b54-d786-4d09-af2e-2459c8260a4a.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUzMjk4OTQsIm5iZiI6MTczNTMyOTU5NCwicGF0aCI6Ii84ODU4MzY0Ny8zOTg5NzI0NjctNjVjZDFiNTQtZDc4Ni00ZDA5LWFmMmUtMjQ1OWM4MjYwYTRhLlBORz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjI3VDE5NTk1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI1OTdiNDcyNDg3ODkwNmI3ZDQxOWE5ZmI0MDFmYjk5NGFkNmM2MTgzYmEwYmMyYmMzYWM1YzY2M2VlYTJlZjcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.tVVA3K_uCl93x5n1wLLhs0XjQ7CnBM69sDQ_CoAxeZ4)
> > I cross checked that the effect is not being overwritten somewhere else by giving a random color like this: ![5th](https://private-user-images.githubusercontent.com/88583647/398972645-437bb1c5-b0b1-4939-873e-4b7c748b93b6.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUzMjk4OTQsIm5iZiI6MTczNTMyOTU5NCwicGF0aCI6Ii84ODU4MzY0Ny8zOTg5NzI2NDUtNDM3YmIxYzUtYjBiMS00OTM5LTg3M2UtNGI3Yzc0OGI5M2I2LlBORz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjI3VDE5NTk1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY3YzQwYzcyODFhZmNhODc4M2RkYmZhZTZmOWE2MTU2NGU1ZDgyZDY1YWYzNDc0NGFiMzM1ZWRjZmUzYzYyMDMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.dq86iWGobGWrDRijZyrf9UW_0Q765LH6JLR6LaIwDNk) ![6th](https://private-user-images.githubusercontent.com/88583647/398972650-737709b3-9b00-406f-91c6-9b6894bc528d.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUzMjk4OTQsIm5iZiI6MTczNTMyOTU5NCwicGF0aCI6Ii84ODU4MzY0Ny8zOTg5NzI2NTAtNzM3NzA5YjMtOWIwMC00MDZmLTkxYzYtOWI2ODk0YmM1MjhkLlBORz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjI3VDE5NTk1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY5NmJkNWRiOTFlOWJmYTU3ODNjOGNhOWMzM2EyNzNlMjBjMzMzOTE3OWNjNjJmMzY3NGZhNmEyNTkwZTM1ODgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0._TkoOpNhvGDEYwOyyW76mdjJFFGoVWIpVfX2VfBmQ-s)
> > So shall I replace the sx prop hardcoded colours with global CSS vars?
> 
> Did you replace the hard coded values? I didn't check

I changed the values in this particular file... but there could be like more files which could be worked for the similar taste.... Shall I raise an issue for such refactoring or just simply make so and so changes in the files?

**Comment by AceHunterr** [comment]
> Also the blue header is too bold. Use the work of he reference PR to select a color

I didn't change any colors though... although the blue header could be lighten a bit shall I simply submit another PR for that?

**Comment by palisadoes** [comment]
1. Yes. Do a global search and replace, preferably using a variable name. The theme is using pastels.
2. What is the hex code for it?

**Comment by AceHunterr** [comment]
Currently the hex code is this : `#0056b3`

`  --bs-primary: #0056b3;
  --table-head-bg: var(
    --bs-primary,
    blue
  );`
  
  The header row of the table has this bg color

**Comment by palisadoes** [comment]
Please let me know what the replacement color you choose will be. Submit the PR any way

**Comment by AceHunterr** [comment]
@palisadoes Also I found some UI abnormalities in the talawa admin dashboard on some parts which feels very weird:

1.Over lapping of components in the organisation cards.
![1](https://github.com/user-attachments/assets/28e474a0-22d4-4a3e-ae90-0495a76e42f5)

2.Profile images not centered in div.
![2](https://github.com/user-attachments/assets/6b371363-6c03-4c34-aa99-4134a3314d02)

3.Again overlapping non uniform lengthfields in the mobile view.
![3](https://github.com/user-attachments/assets/9388b983-0101-4a3e-b4ff-94d74cd82a0f)

4.Sudden disapperance of items on the screen when clicking on Day and Year view.
https://github.com/user-attachments/assets/910cce74-1c8e-4086-9f8f-2fd5b3a6bc6f

5. White hover color over the light background feels very uneasy.
https://github.com/user-attachments/assets/e23b1f44-6e71-4d64-b674-96c119fb63c3

These are a few things I noticed and I tested on 2-3 browsers and they were there... If these can be worked upon... I would like to raise an issue for them as well.

So basically would raising 2 issues... 

one for refactoring the hard coded values with global css root variables for all the files in the code... 
second for rectifying the UI flaws mentioned...

Are they good to go?

**Comment by palisadoes** [comment]
Please fix them all in the PR to change that blue color. Thanks for being observant. If was inevitable we'd see these inconsistencies with the CSS consolidation.

**Comment by AceHunterr** [comment]
> Please fix them all in the PR to change that blue color. Thanks for being observant. If was inevitable we'd see these inconsistencies with the CSS consolidation.

Sure I'll simply raise an issue and sort them.
